src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/profile_2017_03_09/test_dns_zone_import.py

```diff
@@ -192,9 +192,9 @@ def test_zone_file_2(self):
         self._check_txt(zone, 't2.' + zn, [(3600, None, 'foobar')])
         self._check_txt(zone, 't3.' + zn, [(3600, None, 'foobar')])
         self._check_txt(zone, 't4.' + zn, [(3600, None, 'foo;bar')])
-        self._check_txt(zone, 't5.' + zn, [(3600, None, 'foo;bar')])
-        self._check_txt(zone, 't6.' + zn, [(3600, None, 'foo;bar')])
-        self._check_txt(zone, 't7.' + zn, [(3600, None, '"quoted string"')])
+        self._check_txt(zone, 't5.' + zn, [(3600, None, 'foo\\;bar')])
+        self._check_txt(zone, 't6.' + zn, [(3600, None, 'foo\\;bar')])
+        self._check_txt(zone, 't7.' + zn, [(3600, None, '\\"quoted string\\"')])
```

Comment:
@tjprescott is it needed in the `latest` profile's tests?

Reply:
Is DNS part of stack?  I didn't think it was.  While it is in the Network module, DNS is a separate RP.

Reply:
But in regard to the double quotes...possibly. There are some live-test failures or DNS and it might be related to this.

Reply:
Travis, yes DNS is part of stack. But to be clear, we are only recording the tests on Public Azure with 2017-03-09-profile, not stack.